libs/auth/src/apdu.test.ts

```diff
@@ -114,13 +114,13 @@ test('CardCommand with data requiring multiple APDUs', () => {
       Buffer.alloc(110, 3),
     ]),
     Buffer.concat([
-      Buffer.from([
+      Buffer.of(
```

Comment:
This commit finishes up a change started in https://github.com/votingworks/vxsuite/pull/4041